com.unity.netcode.gameobjects/Runtime/NetworkVariable/Collections/NetworkSet.cs

```diff
@@ -207,135 +189,9 @@ public override void ReadDelta(Stream stream, bool keepDirtyDelta)
             }
         }
 
-        /// <inheritdoc />
-        public IEnumerator<T> GetEnumerator()
-        {
-            return m_Set.GetEnumerator();
-        }
-
-        /// <inheritdoc />
-        IEnumerator IEnumerable.GetEnumerator()
-        {
-            return m_Set.GetEnumerator();
-        }
-
-        /// <inheritdoc />
-        public void ExceptWith(IEnumerable<T> other)
```

Comment:
These are operations that the Native containers don't support now nor seem likely to support in the future, and I am loathe to write and support implemenations for all these.  I am assuming these were here to fill out the (now-deleted) ISet<> interface than to service netcode needs.